### ISSUE TITLE
Dashboard: CSS errors

### ISSUE THREAD

**Issue by dwertheimer** [NotePlan/plugins#603]
Rollup continues to complain (and slow down) due to CSS it doesn't understand. I have updated to the latest Rollup and have added rollup modules for postCSS etc. to try to get them to understand it. But no joy.
```
[plugin postcss] jgclark.Dashboard/src/react/css/Dashboard.css: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: l*1.4
^..^
[plugin postcss] jgclark.Dashboard/src/react/css/Dashboard.css: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: l*0.8
^..^
[plugin postcss] jgclark.Dashboard/src/react/css/Dashboard.css: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: l*1.4
^..^
[plugin postcss] jgclark.Dashboard/src/react/css/Dashboard.css: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: l*1.4
^..^
[plugin postcss] jgclark.Dashboard/src/react/css/Dashboard.css: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: l*0.8
^..^
[plugin postcss] jgclark.Dashboard/src/react/css/Dashboard.css: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: l*1.4
^..^
[plugin postcss] jgclark.Dashboard/src/react/css/DashboardDialog.css: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: l*1.4
^..^
[plugin postcss] jgclark.Dashboard/src/react/css/DashboardDialog.css: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: l*1.4
^..^
[plugin postcss] jgclark.Dashboard/src/react/components/Header/Header.css: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: l*0.8
^..^
[plugin postcss] jgclark.Dashboard/src/react/components/Header/Header.css: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: l*0.8
^..^
[plugin postcss] np.Shared/src/react/DynamicDialog/DynamicDialog.css: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: l*0.8
^..^
[plugin postcss] np.Shared/src/react/DynamicDialog/DynamicDialog.css: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: l*0.8
^..^
[plugin postcss] np.Shared/src/react/DynamicDialog/DropdownSelect.css: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: l*1.4
^..^
[plugin postcss] np.Shared/src/react/DynamicDialog/DropdownSelect.css: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: l*1.4
^..^
```

I don't know anything about CSS of this type, so I asked ChatGPT what to do and here's what chat said...the last one (OPTION B) seems to make the most sense to me, and it's easier to understand in the CSS. I went ahead and made the required changes to the Rollup script. You would only need to change the CSS.

![Screen Cap 2024-12-01 at 18 00 39@2x](https://github.com/user-attachments/assets/260f377d-bf5c-44ee-bad7-f7adb944c30d)

![Screen Cap 2024-12-01 at 18 01 06@2x](https://github.com/user-attachments/assets/34d60653-e8d2-45d7-8499-812a3dfef758)

![Screen Cap 2024-12-01 at 18 01 49@2x](https://github.com/user-attachments/assets/fbe7f115-ee9a-473c-acd1-84c21d3cd690)

**Comment by jgclark** [comment]
Note to self: David has tweaked rollup etc. to add the postcss-color-function.

It needs more investigation to work out which of Option A or B is:
- flexible enough
- changes least code
- can be changed back again easily when this new CSS becomes more widely adopted.

**Comment by jgclark** [comment]
I've found a way to fix the CSS without using the extra package. It's not perfect, but I don't want to rely on a CSS processor, as it only adds dependencies and therefore more complexity downstream. The CSS I used will get adopted more widely I'm sure, so I've left the original CSS in the files, to revert to it in future.

I tried removing the extra packages and updates the rollup, but then the build failed. So I've left them there for now, but made notes we should remove them.
